src/views/Register.vue

```diff
@@ -2,9 +2,9 @@
   <div class="flex flex-col justify-center bg-gray-100">
     <div class="mx-8 text-center sm:mx-auto">
       <img
-        src="https://tailwindui.com/img/logos/workflow-mark-indigo-600.svg"
-        alt="Workflow"
-        class="h-12 mx-auto"
+        class="h-10 mx-auto"
+        src="https://www.binarcode.com/img/brand/logo-dark.png"
```

Comment:
Same as above. Also since this repeats 3 times, let's extract an AppLogo component instead